static/sass/_patterns_carousel.scss

```diff
@@ -25,58 +26,106 @@
       text-align: center;
 
       img,
-      video,
-      iframe {
+      video {
         display: block;
-        margin-bottom: 0;
+        margin-bottom: 0 !important;
+        max-height: 460px;
         max-width: 100%;
-        max-height: 362px;
       }
     }
 
-    .p-carousel__item--video {
-      position: relative;
+    // videos holder
+    &--videos {
+      .p-carousel__item-holder {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+
+        .p-carousel__item--screenshot {
+          display: flex;
+          flex-grow: 0;
+          height: auto;
+          margin-bottom: $spv-inter--regular;
+          max-height: 78px;
+          overflow: hidden;
+        }
+
+        .p-carousel__item-screenshot img,
+        .p-carousel__item-screenshot video {
+          align-self: center;
+          width: 100%;
+        }
+
+        @media screen and (max-width: $breakpoint-medium) {
+          flex-direction: row;
+          flex-wrap: wrap;
+          justify-content: flex-start !important;
+          margin-bottom: -$spv-inter--regular;
+          margin-right: -$sph-inter;
+          margin-top: $spv-inter--regular;
+
+          .p-carousel__item--screenshot {
+            margin-bottom: $spv-inter--regular !important;
```

Comment:
So many `!important`s... Is there something that changes the margin, other styles get applied here as well?

Reply:
Updated :D